boat-scaffold/src/test/java/com/backbase/oss/codegen/java/BoatSpringCodeGenTests.java

```diff
@@ -166,267 +166,267 @@ void testReplaceBeanValidationCollectionType() {
                 codegenProperty,"Set<@Valid com.backbase.dbs.arrangement.commons.model.TranslationItemDto>");
         assertEquals("Set<com.backbase.dbs.arrangement.commons.model.@Valid TranslationItemDto>", result);
     }
-    @ParameterizedTest
```

Comment:
why did you change this file?

Reply:
These tests were failing locally. I've uncommented this file.